test/fluid/comments/map.fld

```diff
@@ -0,0 +1 @@
+map ((+) 1) """Takes each ${x} to ${x + 1}""" [4, 6, 12, 14, 3, 2, -4]
```

Comment:
Although this example doesn't really make sense, I guess it's ok for now as it will fail once we start evaluating doc comments!